src/test.rs

```diff
@@ -29,88 +30,88 @@ pub(crate) fn test(args: Args) -> Result<(), Error> {
         test_config
     };
 
-    let mut tests = Vec::new();
-
-    let crate_metadata = metadata
+    let tests = metadata
         .packages
         .iter()
         .find(|p| Path::new(&p.manifest_path) == config.manifest_path)
-        .expect("Could not read crate name from cargo metadata");
-    let target_iter = crate_metadata.targets.iter();
-    for target in target_iter.filter(|t| t.kind == ["bin"] && t.name.starts_with("test-")) {
-        println!("{}", target.name);
+        .expect("Could not read crate name from cargo metadata")
```

Comment:
Note that most of the diff below is due to the indentation changes, almost 99% of the block is the same as before.